internal/docker/config.go

```diff
@@ -53,18 +52,6 @@ func NewDefaultConfig() *Config {
 	return cfg
 }
 
-// validate asserts that an endpoint field is set
-// on the config struct
-func (config Config) validate() error {
-	if config.Endpoint == "" {
-		return errors.New("config.Endpoint must be specified")
-	}
-	if err := VersionIsValidAndGTE(config.DockerAPIVersion, minimumRequiredDockerAPIVersion); err != nil {
-		return err
-	}
```

Comment:
This is already being validated by each components `Validate`: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d70e30c38af6465cfcf9942677b7ae7392066cec/receiver/dockerstatsreceiver/config.go#L50